packages/grafana-data/src/transformations/transformers/histogram.ts

```diff
@@ -38,10 +38,13 @@ export const histogramBucketSizes = [
 ];
 /* eslint-enable */
 
+const DEFAULT_BUCKET_COUNT = 30;
```

Comment:
I think you mentioned the old default was 20? Is 30 enough to make a notable diff?

Reply:
i think the increased resolution helps, and brings it closer to the old graph panel at 25%-50% panel widths, which is probably common (vs 100% histogram panel width)